### PR TITLE
Updated README for Hue binding with fixed example and some clarifications

### PR DESCRIPTION
**Pull request by justClouds**
Corrected the error in the example item definitions (missing bridge ID on some items) and added a little clarification on sensors and switches, and the handling of switch events. There seemed to be some confusing, judging by the comments in the community forum. ;o)

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.hue/README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ The integration happens through the Hue bridge, which acts as an IP gateway to t
 
 ## Supported Things
 
-The Hue bridge is required as a "bridge" for accessing any other Hue devices.
+The Hue bridge is required as a "bridge" for accessing any other Hue devices. It supports the ZigBee LightLink protocol as well as the upwards compatible ZigBee 3.0 protocol. There are two types of Hue bridges, generally refered to as v1 (the rounded version) and v2 (the squared version). Only noticable difference between the two generation of bridges is the added support for Apple HomeKit in v2. Both are bridges fully supported by this binding.
```
Comment (hakan42) [inline extensions/binding/org.eclipse.smarthome.binding.hue/README.md]: Could you please put each sentence on its own line? The formatting is done with the website generator later on.

**Comment by justClouds** [comment]
Sorry about that. Will do.

File: extensions/binding/org.eclipse.smarthome.binding.hue/README.md [hunk 2]
```diff
@@ -253,8 +256,17 @@ sitemap demo label="Main Menu"
...
 rule "example trigger rule"
 when
-    Channel "hue:0820:dimmer-switch:dimmer_switch_event" triggered <EVENT>
+    Channel "hue:0820:1:dimmer-switch:dimmer_switch_event" triggered <EVENT>
 then
     ...
 end
 ```
+
+The optional `<EVENT>` represents one of the button events that are generated by the Hue Dimmer Switch. If ommited the rule gets triggerd by any key action and you can determine the event that triggered it with the `receivedEvent.getEvent()` method. Be aware that currently the events have a '.0' attached to them, like `2001.0` or `34.0`. So, testing for specific events looks like this:_
```
Comment (hakan42) [inline extensions/binding/org.eclipse.smarthome.binding.hue/README.md]: "triggered". Missing an 'e' 😄

**Comment by justClouds** [comment]
> @justClouds Thanks for this PR! Please also note the [contribution guidelines](https://github.com/eclipse/smarthome/blob/master/CONTRIBUTING.md) that require you to have a valid ECA and to sign-off your commits for any non-trivial contribution (as which I regard this PR).

OK. Will do.

**Comment by justClouds** [comment]
Read & signed ECA.

**Comment by kaikreuzer** [comment]
Moved it to https://github.com/openhab/openhab2-addons/pull/4666

**Pull request closed without merging**